docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,651 @@
 
 .. _wallet-instance-attestation.rst:
 
-wallet-instance-attestation.rst
-+++++++++++++++++++++++++++++++
-
-[What is it]
-    
-[What it is usefull for]
-    
-[Example]
-    
+Wallet Instance Attestation
++++++++++++++++++++++++++++
+
+Inside a **Wallet Solution** and, especially with regards
+to the **Wallet Instance**, it is essential to ensure the **authenticity,
+integrity, security, privacy and trust** in the use of the latter both
+by the User and the services connected to it, such as the
+**PID Provider** or one **Relying Party**.
+
+
 General Properties
 ------------------
-    
-[TODO]
 
-    
+The goal is:
+
+- Ensure that the Wallet Instance maintains a level of **integrity**,
+  capable of preventing any attempts of manipulation or forgery
+  by unauthorized third parties.
+- Make sure the private keys have been generated and stored securely
+  within a **trusted execution environment** (TEE).
+- Verify that the Wallet Instance is **authentic**, i.e. made available
+  by an accredited body that complies with the Trust Model.
+
+To guarantee the above, it is necessary for each
+Wallet Instance to issue a certificate of conformity,
+guaranteeing its security and compliance with the Trust Model.
+
+This attestation is called **Wallet Instance Attestation**
+and must be electronically signed by its issuer.
+
+.. hint::
```

Comment:
Could it be useful to specify the wallet provider is the actual leaf in the entity federation tree?